src/home/UnauthenticatedView.tsx

```diff
@@ -48,8 +48,9 @@ export const UnauthenticatedView: FC = () => {
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState<Error>();
   const [optInAnalytics] = useOptInAnalytics();
-  const [privacyPolicyUrl, recaptchaKey, register] =
-    useInteractiveRegistration();
+  const interactiveRegistration = useInteractiveRegistration();
+  const recaptchaKey = interactiveRegistration[1];
+  const register = interactiveRegistration[2];
```

Comment:
maybe refacturing `useInteractiveRegistration` to use an obj would be worthit? wdyt?